tao_compiler/mlir/disc/tests/disc-transform/default_schedule_matmul.cc

```diff
@@ -237,4 +237,22 @@ TEST(PackedMatmul, F32_768x2_Using_Default_Schedule) {
       /*profiling*/ true));
 }
 
+TEST(PackedMatmul, F32_768x3072_Using_Default_Schedule) {
+  EnvSetting setting = {{"DISC_ENABLE_TRANSFORM_SCHEDULE", {"1", false}},
+                        {"DISC_ENABLE_SHAPE_CONSTRAINT_IR", {"1", false}},
+                        {"DISC_MEM_INTENSIVE_OPT_EXPERIMENTAL", {"0", false}}};
```

Comment:
currently `DISC_MEM_INTENSIVE_OPT_EXPERIMENTAL` only works for gpu backend. I just set the flag to avoid run the UT two times (our UT system will enumerate the candidate values of some key flags, and run multiple times).